webroot/adm/client-key.html

```diff
@@ -159,15 +168,26 @@ <h3>Output</h3>
                 return;
             }
 
-            var newClientName = prompt("Enter the new client name", "new client name here");
-            var oldClientName = encodeURIComponent($('#clientName').val());
-            var url = '/api/client/rename?oldName=' + oldClientName + '&newName=' + newClientName;
+            var clientContact = encodeURIComponent($('#clientContact').val());
+            var newClientName = encodeURIComponent($('#clientName').val());
+            var url = '/api/client/rename?contact=' + clientContact + '&newName=' + newClientName;
+            doApiCall('POST', url, '#standardOutput', '#errorOutput');
+        });
+
+        $('#doSetContact').on('click', function () {
+            var newClientContact = prompt("Enter the new contact", "new contact here");
```

Comment:
I'm not sure if we need the placeholder here, I think it might make it more likely to accidentally rename a client to `new contact here`. Could we possibly use `confirm` and just have it fail when there is no new `contact` provided?

Reply:
Also hitting `Cancel` on the `prompt` results in:
![image](https://github.com/IABTechLab/uid2-admin/assets/137864392/2ec3ab6c-988e-4450-a7b3-29d2623d4a58)